Jenkinsfile

```diff
@@ -1,17 +1,21 @@
 pipeline {
-    agent { label 'rosie' }
+    agent none
```

Comment:
I see agent is being overridden below, but can you put a default agent here anyway? So someone doesn't add a new stage and accidentally run on some random node